docs/en/sql-reference/functions/string-functions.md

```diff
@@ -1415,3 +1415,47 @@ Result:
 │ foo                        │
 └────────────────────────────┘
 ```
+
+## formatQuery
```

Comment:
Please put everything into the "other functions" [docs](https://clickhouse.com/docs/en/sql-reference/functions/other-functions).